data/sql_updates/rad_analyst.sql

```diff
@@ -6,7 +16,7 @@ select
     -- I am not sure we want to filter on this, if the name is different here it will behave differently than the other committee searches by name
     cmte_nm as committee_name,
     anlyst_id as analyst_id,
-    rad_branch,
+    fix_party_spelling(rad_branch) as rad_branch,
```

Comment:
Without background to this issue or this data table in general, was there a reason that the misspellings couldn't be corrected via a sql statement?